package.json

```diff
@@ -36,12 +36,12 @@
     "ES2016"
   ],
   "dependencies": {
-    "babel-runtime": "5.6.20",
-    "bluebird": "^2.9.33",
+    "babel-runtime": "^5.8.29",
```

Comment:
The babel-runtime version is automatically bumped by the [`preversion` script](https://github.com/JSRocksHQ/harmonic/blob/81979aee6c8c9ca981a69d694e0fe025ad7d1669/package.json#L88) (which invokes the [`update-babel` script](https://github.com/JSRocksHQ/harmonic/blob/81979aee6c8c9ca981a69d694e0fe025ad7d1669/package.json#L86)). The reasoning for this is explained here: https://github.com/JSRocksHQ/slush-es20xx/issues/30